src/boundary_conditions.jl

```diff
@@ -8,12 +8,11 @@ const solution_fields = (:u, :v, :w, :T, :S)
 const nsolution = length(solution_fields)
 
 abstract type BCType end
-struct Default <: BCType end
-
-abstract type NonDefault <: BCType end
-struct Flux <: NonDefault end
-struct Gradient <: NonDefault end
-struct Value <: NonDefault end
+struct Periodic <: BCType end
+struct FreeSlip <: BCType end
```

Comment:
See above comment: the condition `FreeSlip` is actually a special case of `Flux`.